package.json

```diff
@@ -38,15 +38,16 @@
   "homepage": "https://github.com/appium/WebDriverAgent#readme",
   "devDependencies": {
     "appium-gulp-plugins": "^4.1.0",
+    "appium-support": "^2.29.0",
```

Comment:
it this change deliberate?

Reply:
No. I ran `npm install appium-support --save-dev` and it moved the dependency over to `devDependencies`.... I wish it wouldn't do that.